oetube/aspnet-core/OeTube/Domain/Entities/OeTubeUser.cs

```diff
@@ -22,16 +22,16 @@ private OeTubeUser()
             Name = string.Empty;
             EmailDomain = string.Empty;
         }
-        public OeTubeUser(IdentityUser user):this()
+        public OeTubeUser(IdentityUser user) : this()
         {
             Id = user.Id;
             CreationTime = DateTime.Now;
             SetName(user.UserName);
-            string emailDoman = user.Email.Split("@")[1];
-            Check.Length(emailDoman, nameof(emailDoman),
+            string emailDomain = user.Email.Split("@")[1];
+            Check.Length(emailDomain, nameof(emailDomain),
                          EmailDomainConstants.DomainMaxLength,
                          EmailDomainConstants.DomainMinLength);
-            EmailDomain = user.Email.Split("@")[1];
+            EmailDomain = emailDomain;
```

Comment:
@Beres0 this should not break it, right?

Reply:
Email is mandatory and will be checked during registration. In principle, it shouldn't be a problem.